ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -182,21 +177,23 @@ objects:
         # TODO: the test binary should really be a more structured command - most of these flags should be
         #       autodetected from the running cluster.
         # TODO: bump nodes up to 40 again
-        PATH="/usr/libexec/origin:${PATH}"
-        set -x
-        if [[ -n "${TEST_FOCUS}" ]]; then
-          ginkgo -v -noColor -nodes=${TEST_PARALLELISM} $( which extended.test ) -- \
-            -ginkgo.focus="${TEST_FOCUS}" -ginkgo.skip="${TEST_SKIP}" \
-            -e2e-output-dir /tmp/artifacts -report-dir /tmp/artifacts/junit \
-            -test.timeout=10m ${PROVIDER_ARGS-} || rc=$?
-        fi
-        if [[ -n "${TEST_FOCUS_SERIAL}" ]]; then
-          ginkgo -v -noColor -nodes=1 $( which extended.test ) -- \
-            -ginkgo.focus="${TEST_FOCUS_SERIAL}" -ginkgo.skip="${TEST_SKIP_SERIAL}" \
-            -e2e-output-dir /tmp/artifacts -report-dir /tmp/artifacts/junit/serial \
-            -test.timeout=20m ${PROVIDER_ARGS-} || rc=$?
-        fi
-        exit ${rc:-0}
+        function run-tests() {
```

Comment:
We lost path here which broke installer.  I'll open a PR

Reply:
> We lost path here which broke installer. I'll open a PR

#1814

Reply:
I'm going to clean up the inconsistency and restore the fix in https://github.com/openshift/release/pull/1814